api-reference/beta/api/connector_list_memberof.md

```diff
@@ -1,6 +1,6 @@
 # List memberOf
 
-Retrieve the connectorgroup the connector is a member of.
+Retrieve the connectorGroups the connectors is a member of.
```

Comment:
This should be "Retrieve the connectorGroups this connector is a member of."
Here this SHOULD be connector, because you are looking at the memberships of a specific connector.

**Additionally** it looks like we've changed syntax for specifying ids.  In the past we've put <id> - but this is now changed to {id}.  This isn't critical for this change, but your topics will be inconsistent.  See https://graph.microsoft.io/en-us/docs/api-reference/beta/api/user_get for an example